js/api/Surveys.js

```diff
@@ -18,6 +18,26 @@ export function loadCachedSurveyList() {
   return realm.objects('Survey')
 }
 
+// Finds and returns the list of all surveys cached in Realm.io
+export function loadAllAcceptedSurveys(callback) {
+  loadAcceptedInvitations((err, invitations) => {
+    if (err) {
+      callback(err, [])
+      return
+    }
+
+    let surveys = []
+    const invitationsLength = invitations.length;
+    for (var i = 0; i < invitationsLength; i++) {
```

Comment:
just some more variable declaration cleanup.  surveys could be a `const` and the var inside the loop a `let`, as its reassigned.  acceptedSurvey could be a const below